lib/elixir/lib/kernel/cli.ex

```diff
@@ -360,38 +405,39 @@ defmodule Kernel.CLI do
   # Process commands
 
   defp process_command({:cookie, h}, _config) do
-    if Node.alive? do
-      wrapper fn -> Node.set_cookie(String.to_atom(h)) end
+    if Node.alive?() do
+      wrapper(fn -> Node.set_cookie(String.to_atom(h)) end)
     else
       {:error, "--cookie : Cannot set cookie if the node is not alive (set --name or --sname)"}
     end
   end
 
   defp process_command({:eval, expr}, _config) when is_binary(expr) do
-    wrapper fn -> Code.eval_string(expr, []) end
+    wrapper(fn -> Code.eval_string(expr, []) end)
   end
 
   defp process_command({:app, app}, _config) when is_binary(app) do
     case Application.ensure_all_started(String.to_atom(app)) do
       {:error, {app, reason}} ->
-        {:error, "--app : Could not start application #{app}: " <>
-          Application.format_error(reason)}
+        msg = "--app : Could not start application #{app}: " <> Application.format_error(reason)
+        {:error, msg}
```

Comment:
`msg` pulled out here according to [style suggestions in this issue](https://github.com/elixir-lang/elixir/issues/6643)

Reply:
Beautiful!